### ISSUE TITLE
Something is wrong with the newlines.

### ISSUE THREAD

**Issue by adonig** [NightDiveStudios/shockmac#5]
Something seems to be wrong with the newlines. One can possibly use [sed](https://en.wikipedia.org/wiki/Sed) to fix it.

<img width="1041" alt="screen shot 2018-04-06 at 14 38 41" src="https://user-images.githubusercontent.com/900404/38421700-547f8c24-39a8-11e8-9087-bfd848d1ad8b.png">

**Comment by MajickTek** [comment]
This is normal. They are newlines meant for powerpc macs and are made for compatibility with the ide used to build this game

**Comment by adonig** [comment]
Oh, I see. Hm, Do you think it is possible to bring the GitHub guys to support them? :)

**Comment by bsinky** [comment]
Just curious, what is the newline sequence PowerPC macs used? If it's not CRLF or LF, what is it?

**Edit**: By reading through pull request #2, looks like the line endings in use are just CR.

**Comment by sharkitgood** [comment]
Yeah, given this particular version of the source is being preserved for posterity, @NightDiveStudios may be hesitant to apply changes which, as mentioned in PR #2, could potentially affect the intended build environment.  A note could be added in the readme if it's known to cause problems, perhaps allowing the changes in a separate branch -- though a number of forks that have changed the format do exist already if you are looking to browse the code on the web right away.

They've also mentioned that a cleaned up version they've spent a few months with is coming soon, which will undoubtedly be configured for modern environments.

**Comment by HiPhish** [comment]
+1 for adding a note to the README, otherwise people will keep opening issues and pull requests about this. The problem really lies with the way GitHub renders the code listings and should be fixed there. Sadly GitHub is proprietary software, and so we are at the mercy of the company until someone bothers to look into this particular issue. There are already code hosting sites which are Free Software, this could have been a trivial issue to sort out.
